plugins/output-kafka/src/main/scala/com/stratio/sparkta/plugin/output/kafka/producer/KafkaProducer.scala

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.sparkta.plugin.output.kafka.producer
+
+import java.io.{Serializable => JSerializable}
+import java.util.Properties
+import com.stratio.sparkta.sdk.ValidatingPropertyMap._
+
+import kafka.producer.{KeyedMessage, Producer, ProducerConfig}
+
+import scala.collection.mutable
+
+trait KafkaProducer {
+
+  def send(properties: Map[String, JSerializable], topic: String, message: String): Unit = {
+    val keyedMessage: KeyedMessage[String, String] = new KeyedMessage[String, String](topic, message)
+    KafkaProducer.getProducer(topic, properties).send(keyedMessage)
+  }
+}
+
+object KafkaProducer {
+
+  private val DEFAULT_HOST_PORT: String = "localhost:9092"
```

Comment:
You should use constant with the scala style. Instead of DEFAULT_HOST_PORT -> DefaultHostPort